gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/OrganizationService.java

```diff
@@ -31,6 +31,8 @@ public interface OrganizationService {
 
     OrganizationEntity createOrUpdate(ExecutionContext executionContext, UpdateOrganizationEntity organizationEntity);
 
+    OrganizationEntity update(ExecutionContext executionContext, UpdateOrganizationEntity organizationEntity);
```

Comment:
I find it confusing to have 2 `update` methods with more or less the same arguments but different behavior. Should we rename them as `updateOrganization` and `updateOrganizationAndFlows` for instance?

Reply:
Done